apps/site/src/views/Landing/components/ApplyButton/ApplyButton.tsx

```diff
@@ -0,0 +1,15 @@
+import Button from "react-bootstrap/Button";
+import styles from "./ApplyButton.module.scss";
+
+export default function ApplyButton() {
+	return (
+		<div className={styles.applyButtonContainer}>
+			<Button
+				className={styles.applyButton}
+				href="https://hack.ics.uci.edu/" // TODO Change to application site link
+			>
+				<h1 className={styles.applyButtonText}>Apply!</h1>
```

Comment:
Issue: non-semantic heading usage. Bootstrap provides utility classes to match heading styles, e.g. `.h2`

Reply:
Is this still an issue? I am a bit confused on what you would like me to change.

Reply:
For proper document semantics, Heading elements should be used only for actual headings
> Do not use heading elements to resize text. ([Usage notes, Section Heading elements](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/Heading_Elements#usage_notes))

You could switch switch to something like a `span` element with the `.h2` class